app/models/poster.rb

```diff
@@ -0,0 +1,7 @@
+class Poster < ApplicationRecord
+
+  has_attached_file :avatar, styles: { medium: "300x300>", thumb: "100x100>" }, default_url: "/images/:style/missing.png"
```

Comment:
did you check the hierarchy of these, what comes first, associations, validations or scopes? in the style guides i sent over.